FuelCell/.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Android Debug",
```

Comment:
@CartBlanche 

Can you update these to use this 

```
{
    "name": "Android Debug",
    "type": "maui",
    "request": "launch",
    "preLaunchTask": "maui: Build",
},
```

this will work (providing the Android sln is open). I don't think there should be entires in the launch.json that do not work at all. It will just confuse people (even if the plan is for it to be a place holder). 

Probably do the same for the other PR. 

You can also add an `extensions.json` to the .vscode directory with the following

```
{
    "recommendations": [
        "ms-dotnettools.dotnet-maui",
        "ms-dotnettools.csdevkit",
        "ms-dotnettools.csharp"
    ]
}
```

This will make sure the correct extensions are auto added when the directory is opened.

Reply:
@dellis1972 Please see latest changes.